how-to/register-with-store/package.json

```diff
@@ -24,7 +24,7 @@
     "@types/express": "^4.17.11",
     "@types/node": "^14.14.33",
     "express": "^4.17.1",
-    "openfin-adapter": "^21.64.9",
+    "openfin-adapter": "^21.65.4",
```

Comment:
should this be ^22. like the rest?

Reply:
It should. Though it did work with this version as well:) Fix coming right up.

Reply:
Should I also update all the `package-lock.json` files?

Reply:
Version updated.

Reply:
thanks